rad/monadic/issue-remote.rad

```diff
@@ -244,12 +244,28 @@
           (def c_ (strip-input c))
           (over-issues (... [(@ n) (@ :comments) (@nth i)]) (merge-new c_))))))
 
+
+;; HTML frontend
+
+(def html (ref "QmcENMqxFaYifTCGpHwureKnyEAEfgQXZ8fW35kn17uQPP"))
+
+(def get-html
+  "The frontend."
+  (fn [] (read-ref html)))
+
+(def set-html
+  "The frontend."
+  (fn []
+      (auth [:allowed :admin])
+      (read-ref html)))
```

Comment:
Should this be `write-ref` or is the idea not to be able to actually change it? 
It would be helpful if `get-html` and `set-html` don't have the same doc string.

Reply:
Fixed.